src/UiPatternBase.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drupal\ui_patterns;
+
+use Drupal\Core\Plugin\PluginBase;
+
+/**
+ * Class UiPatternBase.
+ *
+ * @package Drupal\ui_patterns
+ */
+abstract class UiPatternBase extends PluginBase implements UiPatternInterface {
```

Comment:
It's the base class of the Pattern plugin. I think this should stay in the main module.